src/card/card-header.jsx

```diff
@@ -109,12 +109,18 @@ const CardHeader = React.createClass({
       avatar = <Avatar src={this.props.avatar} style={styles.avatar} />;
     }
 
+    const {
+      title,
+      subtitle,
+      ...attrs,
```

Comment:
Just something to consider: in testing we use `attrs` or `attribs` to refer to attributes on the rendered html nodes as opposed to `props` on the react nodes. 

Obviously the internal `attrs` is not visible in testing, but just food for thought.